### PR TITLE
221 contact info card

### PR DESCRIPTION
**Pull request by VladyslavTy**

### CODE REVIEW

File: client/src/modules/detailedView/Contact/ContactSidebar/ContactSidebar.tsx [hunk 1]
```diff
@@ -1,13 +1,137 @@
...
+      >
+        <OrganizationAutocomplete />
+      </EmptyCard>
+    );
+
     return (
       <div className={detailedViewStyles.sidebar}>
-        Sidebar
+        { contactCard }
+        { isEmpty(contact.organization) ? emptyOrganizationCard : organizationCard }
```
Comment (valentyntu) [inline client/src/modules/detailedView/Contact/ContactSidebar/ContactSidebar.tsx]: possibly you don't need isEmpty here, organization cannot be "" or so

File: client/src/modules/lead/EditLead/EditLeadSidebar/EmptyCard/EmptyCard.js [hunk 2]
```diff
@@ -99,11 +99,11 @@ class EmptyCard extends Component {
...
 
   onSave = () => {
-    let updatedLead = {};
-    updatedLead._id = this.props.lead._id;
+    let updatedModel = {};
+    updatedModel._id = this.props.model._id;
     const { item } = this.state;
-    updatedLead[this.props.id.split("-")[0]] = item.id ? item.id : item.name;
-    this.props.onUpdate(updatedLead);
+    updatedModel[this.props.id.split("-")[0]] = item.id ? item.id : item.name;
```
Comment (SamardakOleh) [inline client/src/modules/lead/EditLead/EditLeadSidebar/EmptyCard/EmptyCard.js]: extract `this.props.id.split("-")[0]` to `const modelType`.

File: api/contact/index.js [hunk 3]
```diff
@@ -112,10 +112,24 @@ function createOrganization(name, domain, owner) {
 // @route   PATCH api/contact/:contactId
 // @desc    Update contact
 // @access  Private
-router.patch("/:contactId", validateContactDomain, (req, res) => {
+router.patch("/:contactId", validateContactDomain, async (req, res) => {
   const { hasErrors, errors } = validateContactUpdate(req.body);
   if (hasErrors) return res.status(400).json({ errors });
 
+  let updates = req.body;
+
```
Comment (vladislavkoz) [inline api/contact/index.js]: redundant new line

File: client/src/modules/detailedView/Contact/ContactSidebar/ContactSidebar.test.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        updateContact={noop}
+        addCustomFieldToDomain={noop}
+        organizations={organizations}
+        editCustomFieldInDomain={noop}
+        deleteCustomField={noop}
+      />,
+    );
+    expect(wrapper.find('EmptyCard')).toHaveLength(1);
+  });
+});
```
Comment (vladislavkoz) [inline client/src/modules/detailedView/Contact/ContactSidebar/ContactSidebar.test.tsx]: no new line

File: client/src/modules/detailedView/Contact/ContactSidebar/ContactSidebar.test.tsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    _id: '',
+    custom: [],
+    domain: '',
+    name: '',
+    organization,
+    owner,
+    timestamp: new Date(),
+  };
+
+  const organizations: Organization[] = [
```
Comment (vladislavkoz) [inline client/src/modules/detailedView/Contact/ContactSidebar/ContactSidebar.test.tsx]: use organization wich announced before

File: client/src/modules/detailedView/Contact/detailedContactReducer.ts [hunk 6]
```diff
@@ -1,13 +1,50 @@
...
+};
+
+const contactReducer = (state = initialState, action: Action) => {
   switch (action.type) {
     case LOAD_CONTACT:
       return action.payload;
+    case UPDATE_CONTACT:
+      let updatedContact = { ...state};
+      updatedContact = action.payload;
+      return updatedContact;
```
Comment (alexanderko) [inline client/src/modules/detailedView/Contact/detailedContactReducer.ts]: just return `action.payload`

**Pull request merged**